index.html

```diff
@@ -425,20 +425,13 @@ <h3>Use Cases and Requirements</h3>
 <!-- requirement list start -->
         <ul>
           <li>
-<a>Credentials</a> represent statements made by an <a>issuer</a>.
-          </li>
-          <li>
 <a>Verifiable credentials</a> represent statements made by an <a>issuer</a> in
 a tamper-evident and privacy-respecting manner.
           </li>
           <li>
-<a>Holders</a> assemble collections of <a>credentials</a> and/or
-<a>verifiable credentials</a> from different <a>issuers</a> into a single
-artifact, a <a>presentation</a>.
-          </li>
-          <li>
-Holders transform <a>presentations</a> into <a>verifiable presentations</a>
-to render them tamper-evident.
+<a>Holders</a> assemble collections of <a>verifiable credentials</a> from
+different <a>issuers</a> into a single artifact, a <a>verifiable presentation</a>
+that is tamper-evident.
```

Comment:
```suggestion
different <a>issuers</a> into a single artifact, a <a>verifiable presentation</a>
that might be tamper-evident.
```

Reply:
proof is optional on presentations

Reply:
Proof is optional on both credentials and presentations. Structure of both make verifiability testable, i.e., enable verification, which process might succeed or fail, depending on (among other things) presence of proof.
```suggestion
different <a>issuers</a> into a single artifact, a <a>presentation</a>
that might be tamper-evident (making it a <a>verifiable presentation</a>).
```

Reply:
Rather than accept your suggestions, I've added the word 'can', which I hope addresses your concerns about the strict accuracy of the sentence.

Reply:
a verifiable presentation _can_ be tamper evident, which is all the language currently says.